src/client/components/Logo.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+import { brandText } from '@guardian/src-foundations/palette';
+import { SvgGuardianLogo } from '@guardian/src-brand';
+import { Link } from '@guardian/src-link';
+import { from } from '@guardian/src-foundations/mq';
+import { visuallyHidden } from '@guardian/src-foundations/accessibility';
+
+const svgSize = css`
+  svg {
+    width: 149px;
+    height: 50px;
+    ${from.tablet} {
+      width: 200px;
```

Comment:
how are these sizes at each breakpoint level decided? are they done by eye or set out in DCR?

Reply:
Yeah, that's a really good point. We should align with DCR, let me look into doing that.

Reply:
cool cool

Reply:
This is now done. We should be using the same exact sizes as DCR

Reply:
brill :)